src/utils/_Popper.js

```diff
@@ -171,9 +236,8 @@ Popper.propTypes = {
     referenceComponent: PropTypes.element.isRequired,
     disableEdgeDetection: PropTypes.bool,
     noArrow: PropTypes.bool,
-    placementTargetRef: PropTypes.shape({ current: PropTypes.any }),
     popperClassName: PropTypes.string,
-    popperModifiers: PropTypes.object,
+    popperModifiers: PropTypes.array,
```

Comment:
I don't think this is a breaking change because it's not an exported component and it isn't accessible via exported components.